golang/messages.go

```diff
@@ -119,6 +119,17 @@ type initRes struct {
 
 func (m *initRes) messageType() messageType { return messageTypeInitRes }
 
+// Known transport header keys for call requests.
+const (
+	ArgScheme            = "as"
```

Comment:
Godoc or unexport (you should be getting lint warnings on these).

Also, if you leave them exported, do you want to define a type for CallHeaderName?